Dockerfile

```diff
@@ -2,8 +2,12 @@ FROM continuumio/miniconda3:4.8.2
 LABEL authors="phil.ewels@scilifelab.se,alexander.peltzer@qbic.uni-tuebingen.de" \
```

Comment:
```suggestion
LABEL authors="phil.ewels@scilifelab.se,alexander.peltzer@boehringer-ingelheim.com" \
```

Reply:
Maybe we can upgrade to latest version 4.9 too ?

`FROM continuumio/miniconda3:4.9.2`